server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -173,6 +186,15 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
 
     public handler(rawMessage: IQueuedMessage) {
         let kafkaCheckpointMessage: IQueuedMessage | undefined;
+        const lumberJackMetric = DefaultServiceConfiguration.enableLumberTelemetryFramework ?
+            Lumberjack.newLumberMetric(LumberEventName.DeliHandler) : undefined;
```

Comment:
It should use `this.serviceConfiguration` instead of `DefaultServiceConfiguration`